source/docs/v3/features/vite-bundler.blade.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Vite bundler
+extends: _layouts.documentation
+section: content
+---
+
+# Vite bundler {#vite-bundler}
+
+Enabling the ViteBundler feature makes Vite generate the correct asset paths using the `global_asset()` helper.
```

Comment:
Since this page is so short I'd just add an example of what enabling it means, i.e. uncommenting it in the tenancy config.

With a code example like this:
```php
'features' => [
    // [...]
    Stancl\Tenancy\Features\ViteBundler::class,
],
````

Reply:
https://github.com/stancl/tenancy-docs/pull/209/commits/bfd193f88e94666a0a0273f39747e1c6def3d4a0